### PR TITLE
Added function to remove punctuation from tweets

### PR DESCRIPTION
**Pull request by fspaolo**
Just a simple function to remove undesired characters form the text of the tweets before analyzing them (we are interested in plain words).

### CODE REVIEW

File: DS30.ipynb [hunk 1]
```diff
@@ -149,8 +149,14 @@
     "def nlargest(n, word_scores):\n",
     "    return heapq.nlargest(n, word_scores, key=lambda x: x[1])\n",
     "\n",
+    "def remove_punctuation(text):\n",
+    "    # hardcoded to avoid importing any library\n",
+    "    exclude = '!\"#$%&\\'()*+,-./:;<=>?@[\\\\]^_`{|}~0123456789'\n",
```
Comment (tianhuil) [inline DS30.ipynb:154]: This is a little expansive.  I think numbers, `#` and `@` should definitely be kept (it is twitter after all).

Comment (fspaolo) [inline DS30.ipynb:154]: Ok. I was just thinking on extracting plain english words to make a nice wordcloud or histogram (though I agree some symbols have a meaning/sentiment attached to it).

**Comment by tianhuil** [comment]
I'll close this for now since t doesn't look like there's any action on it.

**Comment by fspaolo** [comment]
Sorry, forgot to update this. I added and push the changes you suggested (keep the symbols '#', '@' and numbers '0123456789') in case this is still useful.

**Pull request closed without merging**